src/pages/index.js

```diff
@@ -25,9 +25,6 @@ const IndexPage = () => (
                                 <h2 className="section-heading">What is freeCodeCamp.org?</h2>
 
                                 <p>
-                                    <a href="https://www.freecodecamp.org/">FreeCodeCamp.org</a> is a supportive group
-                                    of over one million coders who build projects, earn free certificates, and get
-                                    experience by coding for non-profits. Go to{' '}
```

Comment:
I was only meaning for this `<a>` tag to be converted to plain text. Looks like you may have removed the entire sentence by accident.

Reply:
Ah I see what you mean. I must've read the issue incorrectly